pkgs/by-name/sd/SDL2_classic/package.nix

```diff
@@ -211,22 +206,17 @@ stdenv.mkDerivation (finalAttrs: {
     updateScript = nix-update-script {
       extraArgs = [
         "--version-regex"
-        "release-(.*)"
+        "release-(2\\..*)"
       ];
     };
     tests = {
-      pkg-config = testers.hasPkgConfigModules { package = finalAttrs.finalPackage; };
+      pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
       inherit
-        guile-sdl2
-        jazz2
-        SDL2_ttf
-        SDL2_net
-        SDL2_gfx
-        SDL2_sound
-        SDL2_mixer
-        SDL2_image
+        SDL2_classic_ttf
+        SDL2_classic_mixer
+        SDL2_classic_image
         ;
-      inherit (python3Packages) pygame pygame-ce pygame-sdl2;
+      inherit (python3Packages) pygame pygame-ce;
```

Comment:
after https://github.com/NixOS/nixpkgs/pull/407732 this doesn't make much sense anymore. Just drop this line tbh, ideally nothing depends on SDL2_classic anyways.